shared/fs/send-attachment-to-chat/index.desktop.js

```diff
@@ -0,0 +1,66 @@
+// @flow
+import * as React from 'react'
+import * as Types from '../../constants/types/fs'
+import * as Kb from '../../common-adapters'
+import * as Kbfs from '../common'
+import * as Styles from '../../styles'
+import ChooseConversation from '../../chat/conversation-list/choose-conversation'
+import type {Props} from '.'
+
+const SendAttachmentToChat = (props: Props) => (
```

Comment:
if we can have these in one file vs 2 i think we should. there's nothing platform specific about the implementation that requires a different file so I think its better to just have index.js and use Styles.isMobile inside

Reply:
Sounds good; I'll merge them in the next PR.